core/client/fs/src/main/java/alluxio/client/file/RetryHandlingFileSystemMasterClient.java

```diff
@@ -174,21 +174,17 @@ public Long call() throws TException {
 
   @Override
   public synchronized Map<String, alluxio.wire.MountPointInfo> getMountTable() throws IOException {
-    return retryRPC(new RpcCallable<Map<String, MountPointInfo>>() {
-      @Override
-      public Map<String, MountPointInfo> call() throws TException {
-        GetMountTableTResponse result = mClient.getMountTable();
-        Map<String, alluxio.thrift.MountPointInfo> mountTableThrift = result.getMountTable();
-        Map<String, alluxio.wire.MountPointInfo>  mountTableWire = new HashMap<>();
-        for (Map.Entry<String, alluxio.thrift.MountPointInfo> entry :
-            mountTableThrift.entrySet()) {
-          alluxio.thrift.MountPointInfo mMountPointInfoThrift = entry.getValue();
-          alluxio.wire.MountPointInfo mMountPointInfoWire =
-              ThriftUtils.fromThrift(mMountPointInfoThrift);
-          mountTableWire.put(entry.getKey(), mMountPointInfoWire);
-        }
-        return mountTableWire;
+    return retryRPC(() -> {
+      GetMountTableTResponse result = mClient.getMountTable();
+      Map<String, alluxio.thrift.MountPointInfo> mountTableThrift = result.getMountTable();
```

Comment:
Style check failure `RetryHandlingFileSystemMasterClient.java:37:8: error: Unused import - alluxio.wire.MountPointInfo.`
Can you remove line 37 -- the import?